### PR TITLE
Adding ability to scroll to a certain position on the infinite list v…

### PR DESCRIPTION
**Pull request by erenn16**
…iew.  Also add content and frame size getters to add with scrolling calculations.

### CODE REVIEW

File: RefreshInfiniteListView.js [hunk 1]
```diff
@@ -279,6 +279,32 @@ var RefreshInfiniteListView = React.createClass({
             }
         }
     },
+    scrollTo(x,y) {
+      if (this._listView) {
+        var scrollResponder = this._listView.getScrollResponder();
```
Comment (greghe) [inline RefreshInfiniteListView.js]: should be `let`

Comment (erenn16) [inline RefreshInfiniteListView.js]: I'm matching the syntax of the preexisting code, so I switched it to `var`

**Comment by greghe** [comment]
As I recall this component had problems on Android. But I guess we can cross that bridge when we get to it. But we might want to consider an alternative, like [react-native-infinite-scrollview](https://github.com/baspellis/react-native-infinite-scrollview)

File: RefreshInfiniteListView.js [hunk 2]
```diff
@@ -1,317 +1,340 @@
...
-var {
-    Image,
-    View,
-    Text,
-    StyleSheet,
-    ListView,
-    Dimensions,
-    ActivityIndicatorIOS,
-    PropTypes,
+import React from 'react-native';
```
Comment (greghe) [inline RefreshInfiniteListView.js]: We should probably start future proofing this:

`import React from 'react'`;

File: RefreshInfiniteListView.js [hunk 3]
```diff
@@ -1,317 +1,340 @@
...
+let {
```
Comment (greghe) [inline RefreshInfiniteListView.js]: And this would be:

`import {`

File: RefreshInfiniteListView.js [hunk 4]
```diff
@@ -1,317 +1,340 @@
...
+  Image,
+  View,
+  Text,
+  StyleSheet,
+  ListView,
+  Dimensions,
+  ActivityIndicatorIOS,
+  PropTypes,
 } = React;
```
Comment (greghe) [inline RefreshInfiniteListView.js]: And this would be:

`from 'react-native';`

**Comment by greghe** [comment]
👍

**Pull request merged**